### ISSUE TITLE
JSON template / documentation

### ISSUE THREAD

**Issue by JonathanCSmith** [FrancisCrickInstitute/AIoD-Model-Registry#2]
Generate either (or both) a:

- [ ] Manfiest template
- [x] Wiki documentation

**Comment by JonathanCSmith** [comment]
Yeah - wiki for humans / descriptions and pydantic schema output for copy/pasta developers ;) Can probably dump the template in the root folder unless it messes with your manfiest discovery in ai-od?

**Comment by sea-shunned** [comment]
Locally I originally had it to manually skip the Pydantic-generated JSON schema, so not a problem. I removed it as the schema didn't work with e.g. [ajv](https://ajv.js.org/) (in my limited testing), but I can add it back as reference.
